core/storage/email/gae_models_test.py

```diff
@@ -128,9 +149,107 @@ def test_get_by_hash_behavior_with_sent_datetime_lower_bound(self):
                     sent_datetime_lower_bound='Not a datetime object')
 
 
+class ReplyToIdModelUnitTests(test_utils.GenericTestBase):
+    """Test the GeneralFeedbackEmailReplyToIdModel class."""
+
+    def setUp(self):
+        super(ReplyToIdModelUnitTests, self).setUp()
+
+        # pylint: disable=unused-argument
```

Comment:
Please do not disable pylint rules. Isn't there any other way to do this?

Reply:
It's not my code, but I can try to remove it.

Reply:
Looks like the linter is happy with naming the parameter unused_cls and removing the line !